packages/modul-components/src/utils/toast/toast-service.ts

```diff
@@ -66,11 +66,9 @@ export default class ToastService {
 
         toast.offset = toast.isTop ? this.baseTopPosition : '0';
 
-        if (params.text.charAt(0) === '<') {
-            toast.$slots.default = [toast.$createElement(Vue.compile(params.text))];
-        } else {
-            toast.$slots.default = [toast.$createElement('p', params.text)];
-        }
+        params.text = params.text.charAt(0) === '<' ? params.text : `<p>${params.text}</p>`;
```

Comment:
utiliser la réponse de ce stack overflow pour un check plus safe! 

https://stackoverflow.com/questions/15458876/check-if-a-string-is-html-or-not/15458987